src/client/index.ts

```diff
@@ -250,6 +250,30 @@ export class Client {
 		)
 	}
 
+	public evmAvailable() {
```

Comment:
@oleggrib let's make this a single function that takes the blockchain as a parameter. Avoid repetition like so:

```
public hasIssuerForBlockchain(blockchain: 'evm' | 'solana' | 'flow') {
		return (
			this.config.issuers.filter((issuer: OnChainTokenConfig) => {
				// EVM should always be active when we have off-chain attestations as it's used for UN challenge signing
				if (blockchain === 'evm' && !issuer.onChain) return true

				// Defaults to evm if blockchain isn't specified and is an onchain token
				return (issuer.blockchain ? issuer.blockchain.toLowerCase() : 'evm') === blockchain
			}).length > 0
		)
	}
```

Reply:
great update! added it as commit